packages/button/src/button.css

```diff
@@ -21,3 +21,72 @@ governing permissions and limitations under the License.
         border-color: highlight;
     }
 }
+
+@keyframes show-progress-circle {
+    0% {
+        visibility: hidden;
+    }
+
+    100% {
+        visibility: visible;
+    }
+}
+
+@keyframes hide-icons-label {
+    0% {
+        visibility: visible;
+    }
+
+    100% {
+        visibility: hidden;
+    }
+}
```

Comment:
Does https://developer.mozilla.org/en-US/docs/Web/CSS/animation-direction make it so we only need one of these?

Reply:
Nope... Since on page load the animation state for everything is 0. and by applying `backwards` or `reverse` to the animation property it takes the current animation state and pushes it to the respective (0 or 100 state). Since at the start the animation was at 0 for icons (say) then even if I do animation backwards on icons with the `show-progress-circle` animation it will not set them to hidden because it is already in 0% state.

Reply:
Really wanted that to work nicely 😞 Seems like it's _possible_, but the actual complexities of it out weight the benefits. Thanks for checking!